extensions/amp-onetap-google/amp-onetap-google.md

```diff
@@ -44,7 +44,7 @@ After a successful sign-in, [`amp-access`](https://go.amp.dev/c/amp-access) and/
 <table>
   <tr>
     <td width="40%"><strong>data-src (required)</strong></td>
-    <td>The URL of the intermediate iframe.</td>
+    <td>The URL of the intermediate iframe. You can use <a href="https://github.com/ampproject/amphtml/blob/main/spec/amp-var-substitutions.md">URL substitution variables</a> to pass values like <code>ACCESS_READER_ID</code>.</td>
```

Comment:
Thanks for doing this! FMI, I noticed in our codebase that we inconsistently use absolute versus relative URLs when linking between md files, would you say absolute is preferred?

Reply:
Prefer relative URLs so links like these are not broken when viewing locally.

Reply:
This is not broken locally, since the URL is live already.

This gets published to `amp.dev`, and I'm not sure that its builder is smart enough to rewrite relative URLs when they're inside an `<a>`.